src/turtle/env/Dmq.d

```diff
@@ -450,4 +460,27 @@ unittest
         test!("==")(size.records, 1);
         test!("==")(size.bytes, 4);
     }
+
+    // Testing serialization and deserialization of versioned structs.
+    {
+        struct VersionedStruct
+        {
+            const StructVersion = 1;
+            int i;
+            char[] j;
+        }
+
+        VersionedStruct a;
+        a.i = 100;
+        a.j = "HelloWorld";
```

Comment:
That doesn't work in D2.

Reply:
Yeah already fixed it, should be building now.